app/javascript/utils/react-bootloader.jsx

```diff
@@ -53,55 +91,62 @@ const render = (elem, component) => {
     </React.StrictMode>,
     elem,
     () => {
-      setTimeout(() => {
-        elem.classList.add('--hydrated')
-      }, 1)
+      // console.log(Date.now(), 'rendered')
+      elem.classList.add('--hydrated')
     }
   )
 
   const unloadOnce = () => {
     ReactDOM.unmountComponentAtNode(elem)
     document.removeEventListener('turbo:before-render', unloadOnce)
   }
```

Comment:
This code is now no longer used. Was that intentional?

Reply:
Yeah as we now have things that persist (e.g. the ones in the header). We can probably make this more sophisticated, but I think it's ok to remove for now. If we see JS errors as a result, we can do the "more sophisticated" bit sooner, and add flags for whether things should unload or not, but I'm happy to try it without.